src/style.css

```
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.item {
+  width: 200px;
+  height: 403px;
+  box-sizing: border-box;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.item__img {
+  width: 160px;
+  height: 134px;
+  margin: 30px 19px 0;
+}
+
+.item__product {
+  margin: 41px 14px 5px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.item__code {
+  margin: 6px 14px 0;
+
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+}
+
+.item__rating {
+  width: auto;
+  margin: 4px 14px 7px;
+
+  align-items: baseline;
+}
+
+.item__rating,
+.item__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.item__stars {
+  display: flex;
+}
+
+.item__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+
+  border-radius: 0.5px;
+  background: url("./images/star.svg") no-repeat center;
+}
+
+.item__stars .item__star_active:nth-of-type(-n+1),
+.item__stars .item__star_active:nth-of-type(-n+2),
+.item__stars .item__star_active:nth-of-type(-n+3),
+.item__stars .item__star_active:nth-of-type(-n+4),
+.item__stars .item__star_active:nth-of-type(-n+5) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.item__price {
+  box-sizing: border-box;
+  margin: 10px 15px 20px;
+
+  align-items: baseline;
+}
+
+.item__feedback,
+.item__price {
+  font-size: 11px;
+  line-height: 13px;
+}
```

```suggestion
.item__feedback,
.item__price {
  font-size: 11px;
  line-height: 13px;
  font-weight: 400;
}
```
Добавь font-weight: 400; в .item__feedback, .item__price-text, .item__code.